### PR TITLE
Update non-standard to Firefox 44

### PR DESCRIPTION
**Pull request by evilpie**
`__noSuchMethod__` and String.prototype.quote and were removed. Firefox 44 is the current stable however. I will try to keep this more up to date in the future.

### CODE REVIEW

**Comment by zloirock** [comment]
`Object#__noSuchMethod__` was removed it FF44, but `String#quote` - in FF37. Please, update PR.

File: data-non-standard.js [hunk 1]
```diff
@@ -71,8 +71,13 @@ exports.browsers = {
     obsolete: true
   },
   firefox30: {
-    full: 'Firefox 30+',
-    short: 'FF 30+',
+    full: 'Firefox 30',
```
Comment (zloirock) [inline data-non-standard.js]: `'Firefox 30-36'`

**Comment by evilpie** [comment]
Do I have to backfill 37, 38, 39 etc? As I said, this seems totally pointless.

**Comment by zloirock** [comment]
Nope, just 37 (as 37-43) and 44.

**Comment by evilpie** [comment]
Thanks :)

**Pull request merged**